medicai/trainer/nnunet/data/resampling.py

```
@@ -0,0 +1,167 @@
+# Utility: compute new shape after resampling
+# ---------------------------------------------------------------------------
+
+
+def compute_new_shape(
+    original_shape,
+    original_spacing,
+    target_spacing,
+):
+    """Return the spatial shape after resampling (rounded to int)."""
+    factors = compute_zoom_factors(original_spacing, target_spacing)
+    return tuple(max(1, int(round(s * f))) for s, f in zip(original_shape, factors))
```

Ruff (0.15.9)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)